### PR TITLE
🍰 2019/kw20/fix_post_card_design

### PR DESCRIPTION
**Pull request by Tirokk**
> [ulfgebhardt](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-17T21:14:05Z" title="Friday, May 17th 2019, 11:14:05 pm +02:00">May 17, 2019</time>_
_Merged <time datetime="2019-05-21T13:30:45Z" title="Tuesday, May 21st 2019, 3:30:45 pm +02:00">May 21, 2019</time>_
---

## 🍰 Pullrequest

fixed post card design

Before:
image

This PR:
image

Target:
image

### Issues

- [X] fixes https://github.com/Human-Connection/Human-Connection/issues/377

### Todo

- [x] Coverage
- [x] Contribution issue

### CODE REVIEW

File: webapp/components/PostCard/index.vue [hunk 1]
```diff
@@ -1,87 +1,94 @@
...
           />
-        </no-ssr>
-      </div>
+        </div>
+      </no-ssr>
     </template>
   </ds-card>
 </template>
 
 <script>
```
Comment (Tirokk) [inline webapp/components/PostCard/index.vue:78]: > [Tirokk](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-05-18T06:13:41Z" title="Saturday, May 18th 2019, 8:13:41 am +02:00">May 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -95,6 +98,7 @@ export default {
       user: 'auth/user'
     }),
     excerpt() {
+      console.log(this.post)
```


Should be deleted?

**Comment by Tirokk** [comment]
> [codecov[bot]](https://github.com/apps/codecov) **Authored by [codecov[bot]](https://github.com/apps/codecov)**
_<time datetime="2019-05-18T11:25:20Z" title="Saturday, May 18th 2019, 1:25:20 pm +02:00">May 18, 2019</time>_

---


# [Codecov](https://codecov.io/gh/Human-Connection/Human-Connection/pull/622?src=pr&el=h1) Report
> Merging [#622](https://codecov.io/gh/Human-Connection/Human-Connection/pull/622?src=pr&el=desc) into [master](https://codecov.io/gh/Human-Connection/Human-Connection/commit/45694b0f89918035f6a8f4d76459a7764974cb2b?src=pr&el=desc) will **increase** coverage by `0.04%`.
> The diff coverage is `100%`.

[Impacted file tree graph](https://codecov.io/gh/Human-Connection/Human-Connection/pull/622?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master     #622      +/-   ##
==========================================
+ Coverage   20.14%   20.18%   +0.04%     
==========================================
  Files         157      157              
  Lines        2209     2210       +1     
  Branches      105      105              
==========================================
+ Hits          445      446       +1     
  Misses       1740     1740              
  Partials       24       24
```


| [Impacted Files](https://codecov.io/gh/Human-Connection/Human-Connection/pull/622?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [webapp/components/User/index.vue](https://codecov.io/gh/Human-Connection/Human-Connection/pull/622/diff?src=pr&el=tree#diff-d2ViYXBwL2NvbXBvbmVudHMvVXNlci9pbmRleC52dWU=) | `92.31% <ø> (ø)` | :arrow_up: |
| [webapp/pages/post/\_id/\_slug/index.vue](https://codecov.io/gh/Human-Connection/Human-Connection/pull/622/diff?src=pr&el=tree#diff-d2ViYXBwL3BhZ2VzL3Bvc3QvX2lkL19zbHVnL2luZGV4LnZ1ZQ==) | `0% <ø> (ø)` | :arrow_up: |
| [webapp/components/PostCard/index.vue](https://codecov.io/gh/Human-Connection/Human-Connection/pull/622/diff?src=pr&el=tree#diff-d2ViYXBwL2NvbXBvbmVudHMvUG9zdENhcmQvaW5kZXgudnVl) | `77.78% <100%> (+2.78%)` | :arrow_up: |

File: webapp/pages/post/_id/_slug/index.vue [hunk 2]
```diff
@@ -70,30 +70,6 @@
...
-          <ds-tag
-            v-for="tag in post.tags"
-            :key="tag.id"
-          >
-            <ds-icon name="tag" />
-            {{ tag.name }}
-          </ds-tag>
-        </div>
-      </template>
-      <ds-space margin="small" />
```
Comment (Tirokk) [inline webapp/pages/post/_id/_slug/index.vue:96]: > [ulfgebhardt](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-18T13:53:59Z" title="Saturday, May 18th 2019, 3:53:59 pm +02:00">May 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -70,30 +70,6 @@
         :is-shouted="post.shoutedByCurrentUser"
         :post-id="post.id"
       />
-      <!-- Categories -->
-      <ds-icon
-        v-for="category in post.categories"
-        :key="category.id"
-        v-tooltip="{content: category.name, placement: 'top-start', delay: { show: 300 }}"
-        :name="category.icon"
-        size="large"
-      />&nbsp;
-      <ds-space margin-bottom="small" />
-      <!-- Tags -->
-      <template v-if="post.tags && post.tags.length">
-        <ds-space margin="xx-small" />
-        <div class="tags">
-          <ds-icon name="tags" />
-          <ds-tag
-            v-for="tag in post.tags"
-            :key="tag.id"
-          >
-            <ds-icon name="tag" />
-            {{ tag.name }}
-          </ds-tag>
-        </div>
-      </template>
-      <ds-space margin="small" />
```


Duplicate code - probl. from a failed merge

File: webapp/locales/de.json [hunk 3]
```diff
@@ -135,6 +135,14 @@
...
       "name": "Aktiv werden"
     },
+    "delete": {
+      "submit": "Löschen",
+      "cancel": "Abbrechen",
+      "success": "Beitrag erfolgreich gelöscht",
+      "title": "Beitrag löschen",
+      "type": "Beitrag",
+      "message": "Möchtest Du wirklich den Beitrag \"<b>{name}</b>\" löschen?"
+    },
```
Comment (Tirokk) [inline webapp/locales/de.json:145]: > [ulfgebhardt](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-18T13:54:23Z" title="Saturday, May 18th 2019, 3:54:23 pm +02:00">May 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -135,6 +135,14 @@
     "takeAction": {
       "name": "Aktiv werden"
     },
+    "delete": {
+      "submit": "Löschen",
+      "cancel": "Abbrechen",
+      "success": "Beitrag erfolgreich gelöscht",
+      "title": "Beitrag löschen",
+      "type": "Beitrag",
+      "message": "Möchtest Du wirklich den Beitrag \"<b>{name}</b>\" löschen?"
+    },
```


Missing translations - from another pr/commit

**Review by Mogge — APPROVED** [review]
Looks good to me! :+1:

**Pull request closed without merging**